sensor_msgs/Image Image
---
int32[] ObjLabels
float32[] Scores
int32[] Bboxes